code/script.js

```diff
@@ -0,0 +1,68 @@
+const USER = "AschwinSiaila";
+const REPOS_URL = `https://api.github.com/users/${USER}/repos`;
+const nameAndPicture = document.getElementById("nameandpicture");
+const projectsContainer = document.getElementById("projects");
+
+const NameandPicture = () => {
+  const personal_ID = `https://api.github.com/users/${USER}`;
+  fetch(personal_ID)
+    .then((res) => res.json())
+    .then((data) => {
+      global__UserData = data;
+    })
+    .then(() => NameAndPicture());
+};
+
+const NameAndPicture = () => {
+  const picture = global__UserData.avatar_url;
+
+  nameAndPicture.innerHTML += `
+        <img class="profile-pic" src="${picture}" alt="Picture of gitHub-user"/>
+        <div class="profile-name">${USER}</div>
+      `;
+};
+
+const fetchRepositories = () => {
+  fetch(REPOS_URL)
+    .then((res) => res.json())
+    .then((data) => {
+      const technigoRepositories = data.filter((repo) => repo.name.includes("project-") && repo.fork);
+
+      console.log(technigoRepositories);
```

Comment:
It's great that you use console logs! They're really helpful! Don't forget to delete them after you finished with the project :)